### PR TITLE
Assigned Control Data in Init and  updated targets in Run

### PR DESCRIPTION
**Pull request by Tan560**
Please let me know if you see any issues with the way I set the controlData and targets of the motors. If this looks okay with you, I can write a bunch of stories/tasks for the rest of the states.

### CODE REVIEW

**Review by witcpalek — CHANGES_REQUESTED** [review]
I really don't like this ...

Why is the code generator making all of the motors public (and not named correctly)?
Why are we making the state changes attribute public?

**Comment by Tan560** [comment]
The name is just what we have called all of the motors, we can change that if you want. But we just went with camel case for the motors and sensors. How should it be done? I guess is this something we can live with for now? We have a robot coming over in a couple days and I don't know if we can really redo all of this. We can talk more tonight if it is easier.

File: src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualLaunch_State.cpp [hunk 1]
```diff
@@ -34,24 +33,32 @@ using namespace std;
...
 }
 
 void noteManagerbackupManualLaunchState::Init()
 {
-	Logger::GetLogger()->LogData ( LOGGER_LEVEL::PRINT, string ( "ArrivedAt" ), string ( "noteManagerbackupManualLaunchState" ), string ( "init" ) );
-
+	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("noteManagerbackupManualLaunchState"), string("init"));
 	m_genState->Init();
+
+	m_mechanism->Elevator->SetControlConstants(0, *m_mechanism->percentOutput);
```
Comment (witcpalek) [inline src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualLaunch_State.cpp:48]: need to move percentOutput to be a private variable and then access it via an accessor

File: src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualLaunch_State.cpp [hunk 2]
```diff
@@ -34,24 +33,32 @@ using namespace std;
...
+	m_mechanism->launcherAngle->SetControlConstants(0, *m_mechanism->percentOutput);
 }
 
 void noteManagerbackupManualLaunchState::Run()
 {
 	// Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("noteManagerbackupManualLaunchState"), string("run"));
 	m_genState->Run();
+	m_mechanism->frontIntake->Set(TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::BACKUP_FRONT_INTAKE));
```
Comment (witcpalek) [inline src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualLaunch_State.cpp:56]: motor need to be private and accessed via accessors

**Review by witcpalek — CHANGES_REQUESTED** [review]

**Comment by Tan560** [comment]
Update Code Gen and not needed anymore

**Pull request closed without merging**